x-pack/plugins/alerting/server/usage/task.ts

```diff
@@ -179,6 +184,10 @@ export function telemetryTaskRunner(
                   avg_total_search_duration_per_day: dailyExecutionCounts.avgTotalSearchDuration,
                   avg_total_search_duration_by_type_per_day:
                     dailyExecutionCounts.avgTotalSearchDurationByType,
+                  avg_actions_run_duration_by_connector_type_per_day:
+                    dailyActionExecutionAggregations.avgRunDurationByConnectorType,
+                  count_connector_types_by_action_run_outcome_per_day:
+                    dailyActionExecutionAggregations.countRunOutcomeByConnectorType,
```

Comment:
**question:** is the data for these metrics only available via alerting rules or can these two stats be generated within the `actions` plugin telemetry? (https://github.com/elastic/kibana/tree/main/x-pack/plugins/actions/server/usage)

Reply:
Yes, that makes sense indeed.
I think i was too focused on alerting usage :) moving them to actions plugin.

Reply:
Done